pkg/kclient/interface.go

```diff
@@ -107,6 +107,12 @@ type ClientInterface interface {
 	GetOnePodFromSelector(selector string) (*corev1.Pod, error)
 	GetPodLogs(podName, containerName string, followLog bool) (io.ReadCloser, error)
 
+	// port_forwarding.go
+	// SetupPortForwarding creates port-forwarding for the pod on the port pairs provided in the
+	// ["<localhost-port>":"<remote-pod-port>"] format. errOut is used by the client-go library to output any errors
+	// encountered while the port-forwarding is running
```

Comment:
I was having a déjà vu that I have written such a comment in the past. Couldn't figure at that time that it was in `pkg/dev/interface.go` and ended up writing the comment again. 🤣